test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/BasicIntegrationTest/CustomDirective.ir.txt

```diff
@@ -0,0 +1,7 @@
+Document - (0:0,0)
+	NamespaceDeclaration - (0:0,0) - 
+		ClassDeclaration - (0:0,0) -  -  -  - 
+			RazorMethodDeclaration - (0:0,0) -  -  -  - 
+				HtmlContent - (0:0,0) - 
+				Directive - (0:0,0) - test_directive
+				HtmlContent - (15:0,15) - 
```

Comment:
These are marker symbols from the syntax tree. We should nuke these from the IR once code gen. Filed an issue: https://github.com/aspnet/Razor/issues/878